third_party/libunwindstack/include/unwindstack/Unwinder.h

```diff
@@ -167,7 +170,9 @@ class UnwinderFromPid : public Unwinder {
   bool Init();
 
   void Unwind(const std::vector<std::string>* initial_map_names_to_skip = nullptr,
-              const std::vector<std::string>* map_suffixes_to_ignore = nullptr) override;
+              const std::vector<std::string>* map_suffixes_to_ignore = nullptr,
+              const std::map<std::string, std::map<uint64_t /*function_start*/, uint64_t /*size*/>>*
+                  functions_to_stop = nullptr) override;
```

Comment:
The name doesn't feel like correct English. Maybe `functions_to_stop_at`? Or `functions_at_which_to_stop` but that sounds ugly.

Reply:
Done.